docs/components/CommentsButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { useRouter } from "next/router";
+import cn from "classnames";
+import { PlusSmIcon } from "@heroicons/react/outline";
+import { getCommentsState, setCommentsState } from "../lib/comments";
+
+export function CommentsButton() {
+  const comments = getCommentsState();
+  const router = useRouter();
+
+  return (
+    // eslint-disable-next-line react/button-has-type -- Can't set it here because it messes up background color. There's a style that sets the background to transparent that I believe is coming from deep within Nextra.
```

Comment:
need permallink to the line of code it comes from so we can know whether this disable is stale the next time someone comes across this code

Reply:
^ this is may main blocking comment, fwiw, not the top level ones. We shouldn't introduce disables without knowing how/when to enable again

Reply:
Oops, had it in there originally and must have lost it while I was doing science. Thanks, good catch!